frontend-react/src/content/resources/api-programmers-guide/ReportStreamApi.mdx

```diff
@@ -1,6 +1,17 @@
 import { CardGroup, Card, CardBody, CardHeader, ProcessList, ProcessListItem, ProcessListHeading  } from "@trussworks/react-uswds";
+import { Button } from "@trussworks/react-uswds";
+import DOMPurify from "dompurify";
 
-# ReportStream API 
+import site from "../../../content/site.json";
+import { ResourcesDirectories } from "../../../content/resources";
+import { trackAppInsightEvent } from "../../../utils/Analytics";
+
+export const handleClick = () => {
```

Comment:
A potential ticket for the future: Update our USLink and Button components to accept a `tracking` prop that takes the parameters of the `trackAppInsightEvent` function to streamline this as it'll be a reoccurring pattern in the app.